README-ca.md

```diff
@@ -14,7 +14,10 @@
 
 Llista d'idiomes disponibles:
 
-- [العربية](./README-ar.md)
+<ul dir="ltr">
+    <li><a href="./README-ar.md">العربية</a></li>
+</ul>
+
```

Comment:
```suggestion
- [&lrm;العربية](./README-ar.md)
```

Reply:
I was looking for something like that! 
Updated :)